JavaScript/tic-tac-toe/src/components/TicTacToe.tsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from "react";
+import GridButton from "./GridButton";
+import styles from "../styles/tictactoe.module.css"
```

Comment:
I would recommend following the trend of always ending the line with the semicolon